sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/getBatchDetectionResultSample.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createAnomalyDetectorRestClient from "@azure-rest/ai-anomaly-detector";
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to For asynchronous inference, get multivariate anomaly detection result based on resultId returned by the BatchDetectAnomaly api.
+ *
+ * @summary For asynchronous inference, get multivariate anomaly detection result based on resultId returned by the BatchDetectAnomaly api.
+ * x-ms-original-file: specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/examples/GetResult.json
+ */
+async function getDetectionResult() {
+  const Endpoint = "{Endpoint}";
```

Comment:
This looks like part of the long-running polling from detect-batch. Users shouldn't generally need to run this if we can use the `getLongRunningPoller` helper on the initial batch response.